### PR TITLE
Support Nebulas (NAS) transactions

### LINKED ISSUE #235: Add support for Nebulas

### PR DESCRIPTION
**Pull request by ArtemGoryunov**
Fixes #235

### CODE REVIEW

File: platform/nebulas/client.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type Client struct {
+	HTTPClient *http.Client
+	URL        string
+}
+
+func (c *Client) GetTxs(address string, page int) ([]Transaction, error) {
+
+	uri := fmt.Sprintf("%s/tx?a=%s&p=%d",
+		c.URL,
```
Comment (vikmeup) [inline platform/nebulas/client.go]: Can you check Binance or waves client? We have a special method to request data easily

File: platform/nebulas/client.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		address,
+		page)
+	res, err := http.Get(uri)
+	if err != nil {
+		return nil, err
+	}
+
+	defer res.Body.Close()
+
+	if res.StatusCode != 200 {
```
Comment (kolya182) [inline platform/nebulas/client.go]:
```suggestion
	if res.StatusCode != http.StatusOK {
```

File: platform/nebulas/client.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		return nil, fmt.Errorf("http %s", res.Status)
```
Comment (kolya182) [inline platform/nebulas/client.go]:
```suggestion
		return nil, fmt.Errorf("Nebulas : %s", res.Status)
```

File: platform/nebulas/client.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	}
+
+	var response Response
+	if decodeError := json.NewDecoder(res.Body).Decode(&response); decodeError != nil {
+		return nil, decodeError
+	}
+
+	var result []Transaction
+	for _, tx := range response.Data.TxnList {
+		if tx.Type == "binary" {
```
Comment (kolya182) [inline platform/nebulas/client.go]: Can you create const of type operations instead hardcode ?

Comment (ArtemGoryunov) [reply platform/nebulas/client.go]: Done

**Pull request merged**